app/controllers/posts_controller.rb

```diff
@@ -1,36 +1,51 @@
 class PostsController < ApplicationController
   before_action :set_user, only: %i[index show new create]
+  before_action :set_post, only: %i[show destroy]
```

Comment:
Nice job so far😉😉 Kindly add the `load_and_authorize_resource` method in your post controller. This method is a declarative way of defining both the loading of a resource and authorizing a user to perform certain actions on that resource. When used in the post controller, it automatically loads the appropriate resource and authorizes the requested action.
```suggestion
  before_action :set_post, only: %i[show destroy]
  
  load_and_authorize_resource
```

Reply:
Please check i already added the cancancan gem. 
![can](https://github.com/faisalakb/Rails-Blog-App/assets/70527740/cd907230-0218-40d7-acbf-cc3ca990840b)

Reply:
Hello @faisalakb  As you can see from the green line on 76 and 77, The changes were made without being pushed to GitHub. This is a picture of your current gemfile which ends at 75.

<img width="809" alt="image" src="https://github.com/faisalakb/Rails-Blog-App/assets/64643995/35ca101a-14e6-405d-874b-df72753069d8">

Please do well to commit your changes👍🏼.